README.md

```diff
@@ -14,9 +14,21 @@ Edit `LocalPreferences.toml`, set `resultsets_dir` to your ADRIA.jl Outputs (or
 Example:
 
 ```toml
-resultsets_dir = "C:\\Users\\awhite\\code\\ADRIA.jl\\sandbox\\Outputs"
+resultsets_dir = "<path/to/ADRIA-results-dir>"
+```
+
+Create or modify the existing ADRIA `config.toml` file such that ADRIA store new model runs
+in the same directory.
+
+```toml
+[results]
+output_dir = "<path/to/ADRIA-results-dir>"
 ```
 
 ## Running
 
-Run `src/main.jl`
+```julia
+using MADAMEAPI
+
+start_server()
```

Comment:
~Do we need to provide the location of the config file or are we assuming there's only ever one in a consistent location?~

I see the function doesn't use a config at all. 

Where does the `config.toml` referred to above come in? Or is this a future work kind of thing?